WalletWasabi/Hwi/HwiClient.cs

```diff
@@ -62,6 +62,7 @@ private async Task<string> SendCommandAsync(IEnumerable<HwiOption> options, HwiC
 			{
 				throw new OperationCanceledException($"'hwi {arguments}' operation is canceled.", ex);
 			}
+
```

Comment:
Do not separate try/catch, add the comment to the end of the catch line.

Reply:
But we have this in our CONTRIBUTING.md file:

> - Place the comment on a separate line, not at the end of a line of code.

I removed the blank line tough https://github.com/zkSNACKs/WalletWasabi/pull/3689/commits/4097fef8db4923f6768cc5659f0c8e925113b470